samples/CommunityToolkit.Maui.Sample/App.xaml

```diff
@@ -25,9 +26,9 @@
                 <Setter Property="BarTextColor" Value="{StaticResource NavigationBarTextColor}" />
             </Style>
 
-            <Style ApplyToDerivedTypes="True" TargetType="ContentPage">
-                <Setter Property="BackgroundColor" Value="{StaticResource AppBackgroundColor}" />
-                <Setter Property="NavigationPage.BackButtonTitle" Value="" />
+            <Style TargetType="Page" ApplyToDerivedTypes="True">
```

Comment:
Support dark theme for the application, otherwise some controls are not visible, especially on Windows